subprojects/problems/src/main/java/org/gradle/api/problems/internal/DefaultProblemBuilder.java

```diff
@@ -231,7 +229,7 @@ public void report() {
     }
 
     public RuntimeException throwIt() {
-        throw throwError(exception, buildInternal(ERROR));
+        throw throwError(exception, buildInternal(ERROR, false));
     }
```

Comment:
🤔 Maybe this should be removed, as the action-based throwing is in place?

Reply:
Actually, this is unused code, thanks for pointing this out.